### PR TITLE
DEV2-3360: Add access to chat interface

### PR DESCRIPTION
**Pull request by ofekby**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -221,6 +221,18 @@
...
+        "title": "Submit message"
+      },
+      {
+        "command": "tabnine.chat.state",
+        "title": "Chat global state"
+      },
+      {
+        "command": "tabnine.chat.clear-all-conversations",
+        "title": "Clear all tabnine chat conversations"
+      },
```
Comment (yonip23) [inline package.json:235]: Isn't this available to the users in the search bar as well?
maybe add a condition in the "when" field, and set it to true only when the eval is running?

**Review by yonip23 — COMMENTED** [review]
looks good, see my comment

File: package.json (hunk 1 above)
Comment (dimacodota) [reply package.json:235]: agree with @yonip23, another thing - the vscod's practice of exposing apis, is to return an object from the activate function, see an example usage here https://github.com/codota/tabnine-vscode/blob/b4e19e26e2cbd85bfa16aff075975e00d1b13df5/src/languages/java/JavaHome.ts#L14

**Review by dimacodota — APPROVED** [review]
lgtm, see comment

File: package.json (hunk 1 above)
Comment (yonip23) [reply package.json:235]: im not sure i undestand what is the change you want here @dimacodota

Comment (ofekby) [reply package.json]: @dimacodota Chose to avoid refactoring all the code to expose these methods from the activate function, Used when...

**Pull request merged**